packages/engine/tests/units/experiment.rs

```diff
@@ -73,7 +73,7 @@ pub async fn run_test_suite<P: AsRef<Path>>(project_path: P, language: Option<La
     for (experiment_type, expected_outputs) in experiments {
         // TODO: Remove attempting strategy
         let mut outputs = None;
-        let attempts = 2;
+        let attempts = 10;
```

Comment:
Wow this is really not great. We should look at stability soon

Reply:
We had 4 failed attempts with the previous commit. Increasing to `10` doesn't bloat the CI much because only failed experiments will be tested again, but I agree, that we should focus the bug this week.